pyproject.toml

```diff
@@ -20,7 +20,7 @@ authors = [
 ]
 requires-python = ">=3.9,<3.12"
 license = {text = "Apache 2.0"}
-version = "1.2.7"
```

Comment:
1.3.0?
we've made some interface changes here and in the previous

Reply:
good point. changed